### ISSUE TITLE
Option to adjust generated image resolution

### ISSUE THREAD

**Issue by nishtahir** [octref/polacode#63]
It would be nice to be able to adjust the generated image quality for situations where a high-resolution image is desirable.

**Comment by drbenvincent** [comment]
Agreed. This is an excellent extension, but outputs look very blurry on Apple retina displays
